src/main/java/org/sonarsource/scanner/cli/Main.java

```diff
@@ -147,7 +147,11 @@ private static boolean showStackTrace(Throwable e, boolean debug) {
   }
 
   private void suggestDebugMode() {
-    logger.error("Re-run SonarQube Scanner using the -X switch to enable full debug logging.");
+    if (cli.isEmbedded()) {
+      logger.error("Re-run SonarQube Scanner using the debug mode.");
```

Comment:
Wasn't sure if I should not display any log or display a more generic message. I think I would be more in favor of no message. WDYT?

Reply:
Have you seen a situation where having this message would help the user when run through scanner-for-msbuild?

Reply:
Not really honestly.

Reply:
Then I'm in favor of no message too.